src/api/RequestApi.ts

```diff
@@ -39,14 +42,25 @@ const transformInRequestFromSP = (request: IResponseItem): IInRequest => {
       Email: request.supGovLead.EMail,
       text: request.supGovLead.Title,
     },
-  };
+  } as IInRequest;
```

Comment:
Let's get rid of the dangerous "as" keyword.  The compiler will correctly infer which type this is.  By adding "as IInRequest" we will prevent it from catching things if we change the content being returned to not match the IInRequest return type.  Or for updating this function if we change the elements of the type.